### PR TITLE
BarsRatingsChanges

### PR DESCRIPTION
**Pull request by CactusGreen**
Add sorting to BarsRatings and some other changes

### CODE REVIEW

File: BeerOverflowWindowsApp/BarComparers/ComparerByRating.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            if (barData1.Ratings == null && barData2.Ratings == null)
+            {
+                result = 0;
+            }
+            else
+            if (barData1.Ratings == null && barData2.Ratings != null)
+            {
+                result = 1;
+            }
+            else
```
Comment (augisjay) [inline BeerOverflowWindowsApp/BarComparers/ComparerByRating.cs]: Your "else if" here is written in two lines when in 
"BeerOverflowWindowsApp/BarComparers/ComparerByDistance.cs" (code above)
it's in one line. One of these needs correction, i guess.

File: BeerOverflowWindowsApp/BarRating.cs [hunk 2]
```diff
@@ -48,5 +52,45 @@ public void AddBars(List<BarData> barsList)
...
+                    if (SortByDistance)
+                    {
+                        barsData.BarsList.Reverse();
+                    }
+                    SortByTitleDesc = false;
+                    SortByRatingDesc = false;
+                    SortByDistance = !SortByDistance;
+                    break;
+            }
+
```
Comment (augisjay) [inline BeerOverflowWindowsApp/BarRating.cs]: gap

File: BeerOverflowWindowsApp/GetBarList.cs [hunk 3]
```diff
@@ -46,6 +53,7 @@ private List<BarData> CombineLists(List<BarData> primaryList, List<BarData> seco
             return primaryList;
         }
 
+
```
Comment (augisjay) [inline BeerOverflowWindowsApp/GetBarList.cs]: gaps?

**Comment by joklek** [comment]
Nice job. Could we calculate the distance to the bar, when we fetch it? Then we could display it to the user and use it in the sorting without calculating it every time.

**Comment by joklek** [comment]
Made the changes requested by Augisjay, but still think we need to store the distance to the bar in the bar object.

**Comment by CactusGreen** [comment]
User location is supposed to change every time. If we want to display distance to bars, it will be necessary to recalculate it before displaying.

**Comment by joklek** [comment]
Roger that.

**Comment by CactusGreen** [comment]
What still needs to be done to merge this?

**Pull request merged**